src/ProjectTemplates/Web.ProjectTemplates/content/WebApi-CSharp/.template.config/template.json

```diff
@@ -355,7 +355,7 @@
       "type": "parameter",
       "dataType": "bool",
       "defaultValue": "false",
-      "description": "Disable OpenAPI (Swagger) support"
+      "description": "Enable OpenAPI (Swagger) support"
```

Comment:
The property name is `DisableOpenAPI`, so I'd think `Disable OpenAPI (Swagger) support` is correct? I believe the original issue reported may've been referring to the translation being inaccurate (ie. it saying `enable` instead of `disable` in the translation)?

Reply:
![image](https://user-images.githubusercontent.com/9075439/135919570-8ebfaa29-bebf-4913-a39d-42dd94e95035.jpeg)
Property name is incorrect, but i dont had idea for change. 
Property name is disable because default value is false (false-disable Api, true-enable Api)

Reply:
But effect is like on attached image

Reply:
Ah I see, thanks for clarifying!